clang/test/Driver/riscv-cpus.c

```diff
@@ -20,6 +20,17 @@
 // MCPU-SYNTACORE-SCR1-MAX: "-target-feature" "+zicsr" "-target-feature" "+zifencei"
 // MCPU-SYNTACORE-SCR1-MAX: "-target-abi" "ilp32"
 
+// RUN: %clang --target=riscv64 -### -c %s 2>&1 -mcpu=xiangshan-nanhu | FileCheck -check-prefix=MCPU-XIANGSHAN-NANHU %s
```

Comment:
I prefer to one CHECK line for one feature, but it's OK since other processors aren't in this form.

Reply:
Sometimes checking that there is no unrelated feature is important. `linux-cross.cpp` `// DEBIAN_I686_M64-SAME: {{^}} ` provides an example. But here it may not be important.